packages/theme/src/theme/_variables.scss

```diff
@@ -167,6 +167,11 @@ $btn-secondary-color: #fff !default;
 $btn-secondary-bg: $brand-secondary !default;
 $btn-secondary-border: darken($btn-secondary-bg, 5%) !default;
 
+$btn-tertiary-info-color: $st-tropaz !default;
+$btn-tertiary-default-color: $dark-silver !default;
+$btn-tertiary-line-height: 2.5rem !default;
+$btn-tertiary-font-weight: 600 !default;
```

Comment:
This is not in the guideline, can you check with the UX team to get the wanted value and for them to add it in the guideline please ?

Reply:
height: 25px is in guideline.
font weight: 600 is missed in guideline. Mickaël G sent this to me in slack.

Reply:
I will request ux to add this font weight.

Reply:
👍  I added it in the guideline then